intermol/desmond/desmond_parser.py

```diff
@@ -1080,22 +1080,20 @@ def load_box_vector(self, lines, start, end):
 #            start: starting position
 #            end: ending position
 
-        i = start
-        v = np.zeros([3,3])*units.angstroms
-        while (i<end):
-            if self.atom_box_vars[0] in lines[i]:
-                startboxlabel = i-start
-            if ':::' in lines[i]:
-                endlabel = i
+        v = np.zeros([3, 3]) * units.angstroms
+        import pdb; pdb.set_trace()
+        for i, line in enumerate(lines[start:end]):
+            if self.atom_box_vars[0] in line:
+                startboxlabel = i
+            if ':::' in line:
+                endlabel = i + start
                 break
-            i+=1
-        startbox = startboxlabel+endlabel
-        nvec = 0
-        for i in range(startbox,startbox+9):
-            j = (nvec)/3
-            k = (nvec)%3
-            v[j,k] = float(re.sub(r'\s', '', lines[i])) * units.angstrom
-            nvec += 1
```

Comment:
Hard to tell if it's right without running, but if it's printing the same lines, it's probably good (and will be an easy fix later if loses some generalizability)